src/Core/Manifest/ClassManifest.php

```diff
@@ -260,8 +260,9 @@ public function isFlushed()
      *
```

Comment:
`<rant>`All those `*Loader` and `*Manifest` classes are another example of where we really should have uses interfaces and traits to create reusable patterns.`</rant>`

Reply:
Rant has been documented here: https://github.com/silverstripe/silverstripe-framework/issues/10032#issuecomment-972194064